### DISCUSSION TITLE
.Exe file

### DISCUSSION THREAD

**Discussion by Arezaei1984** [thedjnK/AuTerm#10] [General]
Hi
I was refered to use Auterm to upload my AWS file to Laird gateway. by using the Auterm. in setup instruction, it was mentioned ot use Exe file for windows but the Zip file does no have any Exe file.
is it a mistake or I am doing something wrong.  I am runing on windows.
thanks

**Comment by thedjnK** [comment]
There are no zip files for AuTerm releases, tar files are for linux and 7z files are for windows, see https://github.com/thedjnK/AuTerm/releases
